### ISSUE TITLE
Shut off printer

### ISSUE THREAD

**Issue by pigu36** [QIDITECH/QIDI_MAX3#57]
Hello
Anyone solved issue about resuming after lack of energy?
I know to compromise quality printing but sometimes it's a necessary choice.

**Comment by leckert123** [comment]
Do you mean resuming printing after power is lost?  Apparently this was researched by Qidi can't be done on the XMax3 since the bed will not hold position on the Z axis when power goes off.  The bend tends to drop a bit.

**Comment by pigu36** [comment]
Yes I mean it...quditec answered it was only for a quality issue. To solve it could be improved a manually tune for the bed. It's clear quality could be compromise but it's a choice.
